### DISCUSSION TITLE
Feature Request: Allow filling in variables on the fly before sending a request

### DISCUSSION THREAD

**Discussion by liancheng** [darrenburns/posting#93] [Ideas]
I've been using Posting to trigger certain internal testing workflows and so far it works great! However, I found myself often needing to invoke the same request over and over again with different parameters.

My current workflow looks like this:

1. Add variables to a request
1. Edit a `.env` to update variables
1. Open Posting with the `.env` file and send the request
1. Edit the `.env` to update variables again
1. Re-open Posting to send the same request with different parameters.

It would be great if Posting could ask for the values of all variables involved in the current request being invoked before sending the request.

**Comment by darrenburns** [comment]
I'll be improving this workflow in the future. 

IIRC you don't actually need to restart after editing a .env file. Previews and stuff don't update, but the new value should be fetched when you send the request.
